src/EnergyPlus/CommandLineInterface.cc

```diff
@@ -484,14 +484,7 @@ namespace CommandLineInterface {
         state.files.outputSszTxtFilePath = composePath(sszSuffix + ".txt");
         state.dataStrGlobals->outputAdsFilePath = composePath(adsSuffix + ".out");
         state.files.shade.filePath = composePath(shdSuffix + ".csv");
-
-        if (suffixType == "L") {
-            // out/sqlite.er:w
-            state.dataStrGlobals->outputSqliteErrFilePath = fs::path(state.dataStrGlobals->outDirPath.string() + sqliteSuffix + ".err");
-        } else {
-            // if 'D':  out/eplus-sqlite.err
-            state.dataStrGlobals->outputSqliteErrFilePath = composePath(sqliteSuffix + ".err");
-        }
+        state.dataStrGlobals->outputSqliteErrFilePath = composePath(sqliteSuffix + ".err");
```

Comment:
Before filesystem changes:

https://github.com/NREL/EnergyPlus/blob/669aaac766488b51bc0c4fe58e5b1664ee4e61f1/src/EnergyPlus/CommandLineInterface.cc#L438-L442

Reply:
I think the fix is incorrect as a result. It should be this I think no?

```suggestion
        if (suffixType == "L") {
            // out/sqlite.err
            state.dataStrGlobals->outputSqliteErrFilePath = state.dataStrGlobals->outDirPath / fs::path{sqliteSuffix + ".err"};
        } else {
            // if 'D':  out/eplus-sqlite.err
            state.dataStrGlobals->outputSqliteErrFilePath = composePath(sqliteSuffix + ".err");
        }
```

Reply:
(of course, thank you for catching it!)

Reply:
Yes, I agree with the suggested changes. I didn't catch the distinctions between the different suffix style types. I've tested these changes locally and get the following:

`-s L` -> "sqlite.err"
`-s C` -> "eplusSqlite.err"
`-s D` -> "eplus-sqlite.err"

Thanks, @jmarrec for the suggestion. I'll commit these changes and pull in develop one more time.